docs/examples.md

```diff
@@ -0,0 +1,35 @@
+# Examples
+
+!!! Tip "Logfire playground"
+Example projects provide options to try out Logfire without needing to instrument your app in production.
+
+These are working, stand-alone apps and projects that you can clone, spin up locally and play around with to get a feel for the different capabilities of Logfire.
+
+**Got a suggestion?**
+
+If you want to see an example of a particular language or library, [get in touch](help.md).
+
+# Example repositories on GitHub
+
+## JavaScript
+
+### Cloudflare worker example
+
+This example is based on the scaffolding created from `npm create cloudflare@latest`, and uses the [otel-cf-workers package](https://github.com/evanderkoogh/otel-cf-workers) to instrument a Cloudflare Worker and send traces and metrics to Logfire.
+
+[See it on GitHub :material-open-in-new:](https://github.com/pydantic/logfire/tree/main/examples/javascript/cloudflare-worker/){:target="_blank"}
+
+### Express example
+
+This example demonstrates how to use OpenTelemetry to instrument an Express application and send traces and metrics to Logfire.
+
+[See it on GitHub :material-open-in-new:](https://github.com/pydantic/logfire/tree/main/examples/javascript/express/){:target="_blank"}
+
+
+## Python
```

Comment:
Another good example would be https://github.com/pydantic/logfire-demo, although it needs some love, and doesn't work fully without an OpenAI and GitHub token.